test/SlowTests/SchemaUpgrade/Issues/RavenDB_12321.cs

```diff
@@ -27,7 +27,7 @@ public async Task WillThrowSchemaErrorOnBigAndSmallDocumentSchemaAsync(string sc
                 schemaVer = documentDatabase.DocumentsStorage.Environment.Options.SchemaVersion;
             }
 
-            var schemaDir = new DirectoryInfo($"SchemaUpgrade\\Issues\\DocumentsVersion\\schema_{schemaNum}");
+            var schemaDir = new DirectoryInfo($"SchemaUpgrade/Issues/DocumentsVersion/schema_{schemaNum}");
```

Comment:
It will be better if we zip those directories and embed the zip files. At the beginning of the test you can take that embedded zip and unpack it to the desired destination. Check 'LegacySmugglerTests'.

It is also safer to use 'PathSetting', it will unify the path separators for you (not needed when files are embedded)

Reply:
done